cyvlfeat/sift/phow.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import matplotlib.colors as colors
+import dsift
+
+
+def phow(image, verbose=False, fast=True, sizes=[4, 6, 8, 10], step=2, color='gray',
```

Comment:
Don't have a list for the default sizes as lists are mutable, use a tuple -> `(4, 6, 8, 10)`.